zerver/lib/export.py

```diff
@@ -1709,7 +1709,7 @@ def get_realm_exports_serialized(user: UserProfile) -> List[Dict[str, Any]]:
     for export in all_exports:
         exports_dict[export.id] = dict(
             id=export.id,
-            event_time=export.event_time.ctime(),
+            export_time=ujson.dumps(export.event_time),
```

Comment:
I don't understand -- why do we need JSON to send a timestamp?

Reply:
I was looking at this incorrectly.  I realize now that using `dumps` was just a weird way of converting the `datetime` object to a timestamp. Fixed.